src/App.js

```diff
@@ -1,25 +1,156 @@
-import logo from './logo.svg';
 import './App.css';
+import firebase from './firebase';
+import { getDatabase, ref, onValue, push, remove } from 'firebase/database'
+import axios from 'axios';
+import Header from './Header';
+import Quote from './Quote';
+import Form from './Form';
+import Notes from './Notes';
+import Footer from './Footer';
+import { useState, useEffect } from 'react';
 
 function App() {
+
```

Comment:
I like that you have all your states and functions in App.js. Very well organized and easy to follow